### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by yurii-krymskiy**
- [DEMO LINK](https://yurii-krymskiy.github.io/layout_catalog/)
- [TEST REPORT LINK](https://yurii-krymskiy.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
try to add or remove cards, nothing should be broken

if you have any questions, ask them in fe_chat
you will get your answer much faster

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,432 @@
...
+        </section>
+      </div>
+
+
+
+
+    </main>
+
+
+
```
Comment (IvanFesenko) [inline src/index.html:434]:
```suggestion
      </div>
    </main>
```

File: src/styles/blocks/container.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  width: 1024px;
+  justify-content: center;
+}
+
+@media (min-width: 1200px) {
+  .container {
+    width: 1200px;
+    padding: 50px 128px;
+  }
+}
```
Comment (IvanFesenko) [inline src/styles/blocks/container.scss:15]: Use nesting
```suggestion
  justify-content: center;
  
  @media (min-width: 1200px) {
    width: 1200px;
    padding: 50px 128px;
}
}


```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
navigation and Buy button should have hove effect
Navigation changes color to blue, button to white
2023-04-27_16h16_22
2023-04-27_16h16_41

**Pull request closed without merging**